protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/distribution/CommandDistributionRecord.java

```diff
@@ -104,11 +110,10 @@ public UnifiedRecordValue getCommandValue() {
     final var concreteRecordValue = concrecteRecordValueSupplier.get();
 
     // write the record value property's content into a buffer
-    final var storedRecordValue = commandValueProperty.getValue();
     final var recordValueBuffer = new UnsafeBuffer(0, 0);
-    final int encodedLength = storedRecordValue.getEncodedLength();
+    final int encodedLength = storedCommandValue.getEncodedLength();
     recordValueBuffer.wrap(new byte[encodedLength]);
-    storedRecordValue.write(recordValueWriter.wrap(recordValueBuffer, 0));
+    storedCommandValue.write(recordValueWriter.wrap(recordValueBuffer, 0));
 
     // read the value back from the buffer into the concrete record value
     concreteRecordValue.wrap(recordValueBuffer);
```

Comment:
🔧 Reading these changes makes me think we should also change the name of `recordValueBuffer` and `concreteRecordValue` as well as the usage of record value in the comments.